PowerEditor/src/Notepad_plus.h

```diff
@@ -45,6 +45,13 @@
 #include "md5Dlgs.h"
 #include <vector>
 #include <iso646.h>
+#include <chrono>
+
+extern std::chrono::steady_clock::time_point g_nppStartTimePoint;
+extern std::chrono::steady_clock::duration g_nppInitTime;
+extern std::chrono::steady_clock::duration g_pluginsLoadingTime;
+extern std::chrono::steady_clock::duration g_sessionLoadingTime;
+extern std::chrono::steady_clock::duration g_cmdlineParamsLoadingTime;

```

Comment:
Only g_nppStartTimePoint & g_pluginsLoadingTime need to be external.
The others please make them local, as in the first commit.